website/content/partials/http_api_connect_ca_common_options.mdx

```diff
@@ -0,0 +1,71 @@
+#### Common CA Config Options
+
+The following configuration options are ssupported by all CA providers:
```

Comment:
I didn't read this whole section in detail, but I assume you copied it from the config section and edited the names as appropriate.

Reply:
I made a few changes to the descriptions of these options when copying it into the partial file, and tried to sync the changes back to the section in options.mdx as well. See f24e36a and a944bbd for the specific set of changes.